ingestion-beam/src/main/java/com/mozilla/telemetry/util/Json.java

```diff
@@ -83,6 +87,20 @@ public static String asString(Object data) throws IOException {
     return MAPPER.writeValueAsString(data);
   }
 
+  /**
+   * Read a org.json object for JSON validation.
+   */
+  public static JSONObject readJsonObject(InputStream inputStream) {
+    return new JSONObject(new JSONTokener(inputStream));
+  }
+
+  /**
+   * Read an org.json array for JSON validation.
+   */
+  public static JSONArray readJsonArray(InputStream inputStream) {
+    return new JSONArray(new JSONTokener(inputStream));
+  }
+
```

Comment:
These can be removed now, correct?

Reply:
`readJsonObject` is still necessary for loading the schema into the validator object. `readJsonArray` can be removed.

Reply:
I want to avoid all use of `org.json`'s facilities for reading Json. I would prefer that we remove `readJsonObject` and instead materialize `inputStream` to a byte array and use `Json.readValue(bytes, JSONArray.class)` in analogy to what we do in `JSONSchemaStore`.

Reply:
Right, I see. I forgot about casting, although JSONObject does need to be imported into scope somewhere so Jackson can cast to the instance type.

Reply:
How about this: we add a public static method `JSONSchemaStore.readSchema(bytes)` that calls `Json.readValue(bytes, JSONObject.class)`. That way we don't increase the surface area of classes that see JSONObject.

Reply:
It's quite a bit of indirection to avoid the use of `org.json`, but making the JSONSchemaStore handle parsing schemas seems like a logical fit.